src/modules/hivemq/hivemq-container.ts

```diff
@@ -0,0 +1,56 @@
+import * as path from "path";
+
+import { GenericContainer } from "../../generic-container/generic-container";
+import { StartedTestContainer } from "../../test-container";
+import { AbstractStartedContainer } from "../abstract-started-container";
+import { Wait } from "../../wait-strategy/wait";
+
+const START_LOG_MESSAGE_REGEX = /(.*)Started HiveMQ in(.*)/i;
+const HIVEMQ_BASE_PATH = "/opt/hivemq";
+const MQTT_PORT = 1883;
+
+export enum HiveMQDockerImageName {
+  COMMUNITY_EDITION = "hivemq/hivemq-ce",
```

Comment:
I would pin a version here, else one day this module will break for users.

Reply:
Truly indeed. I check the repo to see how the user can also provide a custom image. Since their use case might also only work with a previous version.

Reply:
Removed `enum`